roles/pulp3-redis/vars/Fedora.yml

```diff
@@ -2,4 +2,3 @@
 pulp_preq_packages:
   - python3
   - python3-devel
-pulp_python_interpreter: /usr/bin/python3
```

Comment:
Already defined in the "pulp" role, so I don't think it's necessary to copy here?

Reply:
I tried it out and I think you're right.